tests_e2e/pipeline/pipeline.yml

```diff
@@ -106,17 +107,27 @@ jobs:
       - task: AzureKeyVault@2
         displayName: "Fetch connection info"
         inputs:
-          azureSubscription: 'azuremanagement'
-          KeyVaultName: '$(connection_info)'
+          azureSubscription: $(connection_info)
+          KeyVaultName: 'waagenttests'
           SecretsFilter: '*'
 
+      - task: AzureCLI@2
```

Comment:
can this be done with AzureKeyVault@2 instead?

Reply:
AzureKeyVault outputs everything into strings. Then we need to write a script to generate into perm/pfx file

Reply:
so download/import to file can't be achieved with AzureKeyVault. You still need to write bash script or cli to import into the file